City-of-Helsinki/drupal-hdbt
# UHF-6840: Add showing school results on map

# [UHF-6840](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6840)
Enable users to view search results on map.

## What was done
In this PR: 
 
* Add button for showing embed of service map containing all units in the search results
* Add some styles for search results top area

## How to install & test:

Check info here: https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/315


[UHF-6840]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-6840?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
